File: Wolf/app/service/mongo_service.js
const Service = require('egg').Service;

class MongoService extends Service {
    async find(alians = [
        `id`, `url`, `topic`, `digest`, `thumb`, `keywords`, `time`
    ], query = { state: 0 }, limit = 10, sort = { 'createTime': 1 }) {
        const { app } = this;
        const data = await app.mongo.find('wolf', {
            query, limit, sort
        });
        return { data };
    }
    async insert(doc = {}) {
        const { app } = this;
        const { result } = await app.mongo.insertOne('wolf', {
            doc
        });
        let data = false;
        if (result.ok === result.n) {
            data = true;
            return { data };
        }
        return { data };
    }
    async insertArray(docs = []) {
        const { app } = this;
        const { result } = await app.mongo.insertMany('wolf', {
            docs,
        });
        let data = false;
        if (result.ok > 0) {
            data = true;
            return { data };
        }
        return { data };
    }
    async update(update = {}, filter = {}) {
        const { app } = this;
        update = { $set: update };
        const { result } = await app.mongo.updateMany('wolf', {
            filter, update,
        });
        let data = false;
        if (result.ok > 0) {
            data = true;
            return { data };
        }
        return { data };
    }
}

module.exports = MongoService;

File: Fox/service/mongoService.py
import pymongo

DBClient = pymongo.MongoClient('mongodb://127.0.0.1:27017/')
DBName = "wolf"
ColName = "wolf"
DB = DBClient[DBName]
Col = DB[ColName]


def find(project={"url": 1, "content": 1}, query={"state": 1}, limit=10, sort={"key": "createTime", "sort": -1}):
    return list(Col.find(query, project).limit(limit).sort(sort["key"], sort["sort"]))


def update_one(query, values):
    if not (query or values):
        return False
    return Col.update_one(query, {"$set": values}).modified_count > 0
